### PR TITLE
.github: Add pull request template

### PR DESCRIPTION
**Pull request by mgorny**
The idea is to increase awareness of the AI policy, as well as other rules, and to inform users *before* they submit a PR.

### CODE REVIEW

**Comment by mgorny** [comment]
Roughly, it looks like this when opening the PR:

Screenshot showing the template automatically included in new pull request form, with the description inferred from the commit above it

Screenshot showing the preview of the description, with checkboxes

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!-- Please put the pull request description above -->
```
Comment (ulm) [inline .github/pull_request_template.md:1]: Add our standard copyright header?

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+---
+
+Please check all the boxes that apply:
+
+- [ ] I can submit this contribution in agreement with the [Copyright Policy](https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin).
+- [ ] This contribution has not been created with the assistance of Natural Language Processing artificial intelligence tools, in accordance with [AI policy](https://wiki.gentoo.org/wiki/Project:Council/AI_policy).
+- [ ] I have certified the above via adding a `Signed-off-by` line to *every* commit in the pull request.
```
Comment (ulm) [inline .github/pull_request_template.md:9]: Flip the second and third line? `Signed-off-by` certifies copyright policy but not AI policy.

**Review by ulm — COMMENTED** [review]
I am still sceptical about adding this to the repository itself. At the very least it should be excluded from distribution via rsync.

File: .github/pull_request_template.md (hunk 2 above)
Comment (mgorny) [reply .github/pull_request_template.md:9]: My assumption was that AI policy is implied by copyright policy but makes sense, will do.

File: .github/pull_request_template.md (hunk 1 above)
Comment (ulm) [reply .github/pull_request_template.md:1]: I mean, if the thing emphasises our copyright policy, then it should set a good example and have a header. :)
Unless that's technically impossible, but I suppose there must be some comment syntax that would hide it from being displayed?

**Pull request closed without merging**